addons/delivery/models/sale_order.py

```diff
@@ -51,8 +51,12 @@ def set_delivery_line(self, carrier, amount):
 
     def action_open_delivery_wizard(self):
         view_id = self.env.ref('delivery.choose_delivery_carrier_view_form').id
+        if(self.env.context.get('recompute')):
```

Comment:
```suggestion
        if(self.env.context.get('recompute')):
            name = _('Update shipping cost')
        else:
            name = _('Add a shipping method')
        return {
            'name': name,
```
shouldn't be clearer this way? @mart-e

Reply:
@mreficent it is clearer but mostly it is correct.
@pch-odoo never put variables inside a `_()` call, otherwise, the terms are not exported